src/main/java/org/utplsql/api/outputBuffer/OutputBufferProvider.java

```diff
@@ -42,9 +42,13 @@ public static OutputBuffer getCompatibleOutputBuffer(Version databaseVersion, Re
 
     private static boolean hasOutput( Reporter reporter, OracleConnection oraConn ) throws SQLException {
 
-        String sql = "select is_output_reporter from table(ut_runner.get_reporters_list) where regexp_like(reporter_object_name, ?)";
+        String sql = "select is_output_reporter " +
+                " from table(ut_runner.get_reporters_list)" +
+                " where ? = substr(reporter_object_name, length(reporter_object_name)-?+1)";
         try ( PreparedStatement stmt = oraConn.prepareStatement(sql)) {
```

Comment:
It would be much better to have a plsql seperate check for it.
The check could be:
```sql
begin
declare
   l_result boolean;
begin
   execute immediate '
   begin 
:x :=' || DBMS_ASSERT.SQL_OBJECT_NAME( ? ) || '() is of ut_output_reporter_base;
   end;'
using out l_result;
exception when others then null;
end;
? := l_result;
end;
```